src/app/home/home.page.html

```diff
@@ -13,7 +13,7 @@ <h3>Samples</h3>
                         <ion-slides>
```

Comment:
Is there a reason you decided to use slides? The other two images don't play any role, and I think the interface and code would be cleaner without them. 
I think you should have a separate page for viewing the uploaded image, as well as the results of its analysis.